src/main/java/org/sagebionetworks/bridge/exporter3/results/ArcResultSummarizer.kt

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.exporter3.results
+
+import org.sagebionetworks.bridge.rest.model.Assessment
+
+class ArcResultSummarizer(val assessment: Assessment) : AssessmentSummarizer {
+
+    init {
+        assert(assessment.frameworkIdentifier == FRAMEWORK_IDENTIFIER)
```

Comment:
What happens if this assert fails? Does it throw a RuntimeException? Should we instead throw a PollSqsWorkerBadRequestException so that SQS doesn't try to serve the request again?